packages/shared-ux/chrome/navigation/src/ui/navigation.tsx

```diff
@@ -18,31 +18,44 @@ import {
 } from '@elastic/eui';
 import React from 'react';
 import { getI18nStrings } from './i18n_strings';
-import { NavigationBucketProps, NavigationProps } from '../../types';
+import type { ChromeNavigationViewModel } from '../../types';
 import { NavigationModel } from '../model';
 import { useNavigation } from '../services';
 import { ElasticMark } from './elastic_mark';
 import './header_logo.scss';
-import { NavigationBucket } from './navigation_bucket';
-
-export const Navigation = (props: NavigationProps) => {
-  const { loadingCount, activeNavItemId, ...services } = useNavigation();
+import { NavigationBucket, type Props as NavigationBucketProps } from './navigation_bucket';
+
+interface Props extends ChromeNavigationViewModel {
+  /**
+   * ID of sections to highlight
+   */
+  activeNavItemId?: string;
+}
+
+export const Navigation = ({
+  platformConfig,
+  navigationTree,
+  homeHref,
+  linkToCloud,
+  activeNavItemId: activeNavItemIdProps,
+}: Props) => {
+  const { loadingCount, activeNavItemId, basePath, navIsOpen, navigateToUrl } = useNavigation();
   const { euiTheme } = useEuiTheme();
 
-  const activeNav = activeNavItemId ?? props.activeNavItemId;
+  const activeNav = activeNavItemId ?? activeNavItemIdProps;
```

Comment:
I am not sure I understand why we can pass the `activeNavItemId` in both the context and the prop. Seems like the context should be enough.

Reply:
++ It probably should have just been in prop to the NavigationProvider in the original code, and passed down through context. In this early phase, having that prop provides control over which collapsible group to pre-expand in the side nav UI, which helps.

Reply:
Ok. We'll refactor once we have the active nav item logic from the Chrome service. 👍